jobs/V2/f1-j1-getPrimeroCasesV2.js

```diff
@@ -16,64 +16,87 @@ fn(state => ({ ...state, data: {}, references: [], referralIds: [] }));
 // GET Primero cases with oscar referrals
 // User Story 1: Generating government referrals
 // #1 - Request oscar referrals only ===========================================
-getCases(
-  {
-    remote: true,
-    last_updated_at: state => `${state.cursor}..`,
-    //last_updated_at: '2023-08-24T00:00:00.862Z..', //for troubleshooting
-    //last_updated_at: state => `${state.cursor}..2023-08-14T15:00:00.862Z`,  //for troubleshooting
-    page: 1,
-    per: 10000,
-    // These cases have been recently updated and MIGHT have a new referral to send to Oscar; we check & filter below
-    // workflow: 'referral_to_oscar', //REMOVED July '23 bc we should rely on services, not case-level statuses
-  },
-  { withReferrals: true },
-  state => {
-    const oscarRefs = state.data;
-    const referralIds = oscarRefs
-      .map(c =>
-        c.referrals
-          .map(x => {
-            console.log('referral:', 'id:', x.id, x.status, x.created_at, x.service_record_id);
-            return x;
-          })
-          .filter(r => new Date(r.created_at) >= new Date(state.cursor)) //Only send referrals created after the last sync
-          .filter(r => r.status === 'in_progress') // And only send referrals where status 'in progress'
-          .map(r => r.service_record_id)
-      )
-      .flat();
-
-    console.log('Detected referral Ids:', referralIds);
-
-    // we filter the services_section in each oscarRef to return only the services that have/have not been sent to Oscar
-    const sentOscarRefs = oscarRefs
-      .filter(c => c.services_section)
-      .map(c => ({
-        ...c,
-        services_section: c.services_section.filter(service => {
-          if (referralIds.includes(service.unique_id)) {
-            // console.log('Detected service to refer to oscar :', service.unique_id);
-            // console.log('Service response type :', service.service_response_type);
-            return true;
-          }
-          // console.log('N/A, service not recently referred to Oscar :', service.unique_id);
+fn(async state => {
+  const perPage = 100;
+  let currentPage = 1;
+  let totalItems = 0;
+  let allCases = [];
+
+  do {
+    console.log('Fetching cases and their referralls...');
```

Comment:
Can we add the page offsets? It'll just help with debugging

Also `referrals` is 2 `r`s, one of everything else :exploding_head:

Reply:
I have fixed the typo, and regarding offset, that is the `currentPage` value and a user will have to update job code to specify which page their fetching next

Reply:
I'm not sure I understand. This is what I mean:

```
console.log(`Fetching page ${currentPage} of cases and their referrals...');
```

Current page starts at 1 and is updated in this for loop?

Reply:
Ooh okay got you, i have updated the `console.log`